Development environment setup/WSL Setup.md

```diff
@@ -0,0 +1,400 @@
+There are several ways of setting up your dev environment on Windows OS. The one we opted to showcase here is using the Windows Subsystem for Linux (we'll use WSL2 or just WSL through this document).
+Under the hood is a plain Ubuntu 20.04, so you should be able to set up the entire LEMP stack without a problem.
+
+There are two ways you can try to set up the dev environment on WSL, one is more foolproof, but takes a bit longer, the other is by just trying to set up [valet-wsl](https://github.com/valeryan/valet-wsl). It is faster, but we've had issues with it in the past.
+
+## Installing the required tools
+
+### WSL2
+
+Make sure you have either Windows 10 or 11 installed.
+
+For Windows 10, make sure you have:
+
+- For x64 systems: version 1903 or higher, with build 18362 or higher
+- For ARM64 systems: version 2004 or higher, with build 19041 or higher
+- Builds lower than 18362 do not support WSL2, please update your Windows.
+
+To check the version and build number, either open up a start menu and in the search bar, search for: `winver`, or open a run window (Windows key + R) and type `winver`.
+
+Make sure you have virtualization enabled in your BIOS, otherwise WSL won't work as intended.
+
+**Note**
+
+In some cases, you might have to enable certain Windows features on. Most notably you should have `Virtual Machine Platform` and `Windows Subsystem for Linux` enabled in the Windows features list (they are located in the Control Panel -> Programs -> Turn Windows features on or off link).
+
+---
+
```

Comment:
```suggestion
In the case WSL is already installed you can check the list of available distros with `wsl --list --online` and you can run

```bash
wsl --install -d <DistroName>
```

to install a desired Linux distro.
```

Reply:
I wouldn't change the order of this, it's basically the same as in the official docs for installing WSL